data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -97,7 +97,11 @@
         <bootproto>dhcp</bootproto>
         <bridge>yes</bridge>
         <bridge_forwarddelay>0</bridge_forwarddelay>
+        % if ($check_var->('SUT_NETDEVICE', 'eth1') or $check_var->('SUT_NETDEVICE', 'em1')) {
+        <bridge_ports><%= $get_var->('SUT_NETDEVICE') %></bridge_ports>
+        % } else {
         <bridge_ports>eth0 eth1 em1 em2</bridge_ports>
+        % }
```

Comment:
If 'SUT_NETDEVICE' is either 'eth1' or 'em1,' then the bridge port is set to 'SUT_NETDEVICE' via get_var. Otherwise, it is set to 'eth0 eth1 em1 em2.'

Looks good can we add an explanation

Reply:
It seems the comment added can do it.